x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -138,6 +144,9 @@ export function WorkspacePanelWrapper({
   const maxDimensions = displayOptions?.maxDimensions;
   const minDimensions = displayOptions?.minDimensions;
 
+  const [resizeRef, setResizeRef] = React.useState<Element | null>(null);
+  const { width, height } = useResizeObserver(resizeRef, 'width');
```

Comment:
Is the `'width'` required here?

```suggestion
  const { width, height } = useResizeObserver(resizeRef);
```